src/components/nature-of-business/NatureOfBusiness.vue

```diff
@@ -113,7 +113,6 @@
                 <v-icon small color="primary">mdi-pencil</v-icon>Change</v-btn>
```

Comment:
Note that dropdown button remains as "Change" (since we're changing an edit)... confirmed with Yui.